docs/vm/cpu-pinning.md

```diff
@@ -0,0 +1,86 @@
+
+### How the Shared CPU Pool is Calculated ?
+
+Harvester reserves CPU resources for system-level operations based on the [GKE formula](https://cloud.google.com/kubernetes-engine/docs/concepts/plan-node-sizes#cpu_reservations), with [System Reserved](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved) and [Kube Reserved](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved) allocated in a 2:3 ratio.
+
+For instance, on a node with 16 CPU cores, this translates to:
+```
+systemReserved: 408 millicores
+kubeReserved: 612 millicores
+```
+This leaves ~15 cores (14980 millicores) available for workloads.
+
+When a VM (pod) requests 4 CPUs with Guaranteed QoS, those 4 cores are dedicated to it. The remaining pods that are not in Guaranteed QoS class share the remaining CPU pool (~11 cores).
+
+## How to Enable/Disable CPU Manager ?
+
+Enabling the CPU Manager sets the CPU manager policy to static. Disabling the CPU Manager switchs the policy back to the none policy. This action must be performed individually on each node.
+
+1. Navigate to the Hosts page.
+2. Click the hamburger button.
+3. Select either Enable CPU Manager or Disable CPU Manager.
+4. Wait ~1 minute for the action to complete.
+
+![enable-cpu-manager](/img/v1.4/cpu-pinning/enable-cpu-manager.png)
+![disable-cpu-manager](/img/v1.4/cpu-pinning/disable-cpu-manager.png)
+
+### Limitations
+
+- Only one master node can enable or disable the CPU Manager at a time. Note that the witness node cannot perform these actions.
+- You must wait for an operation (enable/disable) to finish before starting another on the same node.
+- Any VMs with CPU pinning enabled must be stopped before disabling CPU Manager on that node.
+
+## How to create a VM with CPU Pinning ?
+
+1. Ensure at least one node enable CPU Manager.
+2. Go to Virtual Machines page, click Create. 
+3. Click Advanced Options.
+![create-vm](/img/v1.4/cpu-pinning/create-vm.png)
+4. Scroll down to the bottom, check the Enable CPU Pinning checkbox.
+![vm-advanced-options](/img/v1.4/cpu-pinning/vm-advanced-options.png)
+
+Enabling the checkbox adds `dedicatedCpuPlacement: true` to `.spec.template.spec.domain.cpu` in the virtual machine YAML. In Harvester, when we detect that `dedicatedCpuPlacement` is set to true in the virtual machine definition, we automatically set the CPU and memory resource requests equal to the limits to ensure it meets the criteria for Guaranteed QoS.
+
+- To enable CPU pinning for an existing VM, restart the VM after enabling this option to apply the change.
+- CPU and memory [Resource Overcommit](./resource-overcommit.md) does not apply to VMs with CPU pinning, as they use Guaranteed QoS, meaning their requests and limits are the same.
```

Comment:
```suggestion
Enabling CPU pinning adds `dedicatedCpuPlacement: true` to `.spec.template.spec.domain.cpu` in the virtual machine configuration (YAML). When `dedicatedCpuPlacement` is set to `true`, the CPU and memory resource requests are automatically set to match the limits to ensure that the criteria for Guaranteed QoS are met.

Because the requests and limits are identical, CPU and memory [Resource Overcommit](./resource-overcommit.md) settings do not apply to VMs with CPU pinning enabled.

:::note

To use CPU pinning on an existing VM, you must restart the VM after enabling the feature and saving the change.

:::
```